app/services/sparqls_count_service.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'lodqa/anchored_pgps'
+require 'lodqa/graph_finder'
+require 'sparql_client/cacheable_client'
+
+# returns the count of sparqls
+module SparqlsCountService
+  class << self
+    # Handle sparqls count.
+    def sparqls_count param
+      parallel = 16
+      sparql_client = SparqlClient::CacheableClient.new(param.endpoint_url,
+                                                        parallel, param.endpoint_options)
+      graph_finder = Lodqa::GraphFinder.new(sparql_client,
+                                            param.graph_uri, param.graph_finder_options)
+
+      sparqls = []
+      anchored_pgps = Lodqa::AnchoredPgps.new param.pgp, param.mappings
+      anchored_pgps.each do |anchored_pgp|
+        to_sparql(anchored_pgp, graph_finder) { |sparql| sparqls << sparql }
```

Comment:
これぐらいなら展開して

```suggestion
        graph_finder.sparqls_of(anchored_pgp) do |_bgp, sparql|
            sparqls << sparql
        end
```
と直接書いてもいいかもしれません。

Reply:
対応しました。